bundles/binding/org.openhab.binding.omnilink/src/main/java/org/openhab/binding/omnilink/internal/OmniLinkGenericBindingProvider.java

```diff
@@ -75,12 +75,7 @@ public void processBindingConfiguration(String context, Item item,
 				new Object[] { config.getObjectType(), config.getNumber() });
 		addBindingConfig(item, config);
 
-		Set<Item> items = contextMap.get(context);
-		if (items == null) {
-			items = new HashSet<Item>();
-			contextMap.put(context, items);
-		}
-		items.add(item);
+		super.processBindingConfiguration(context, item, bindingConfig);
 	}
```

Comment:
super.processBindingConfiguration was never called, but method attempted to achieve equivalent result by hand.